client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -55,7 +55,9 @@ class PeopleListSectionHeader extends Component {
 				{ siteLink && (
 					<Button compact href={ siteLink } className="people-list-section-header__add-button">
 						<Gridicon icon="user-add" />
-						{ translate( 'Invite', { context: 'Verb. Button to invite more users.' } ) }
+						<span className="people-list-section-header__button-text">
```

Comment:
Did you mean to leave this change in here? Should this be a separate PR?

Reply:
It wasn't an accident, but I agree, it's better to be in a separate PR.